runtime/compiler/env/j9method.cpp

```diff
@@ -1456,13 +1456,20 @@ TR_ResolvedRelocatableJ9Method::isInterpreted()
    {
    bool alwaysTreatAsInterpreted = true;
 #if defined(TR_TARGET_S390)
-   alwaysTreatAsInterpreted = false;
+   TR::Compilation *comp = TR::comp();
+   if (comp && comp->compileRelocatableCode() && comp->getOption(TR_UseSymbolValidationManager))
+      alwaysTreatAsInterpreted = true;
+   else
+      alwaysTreatAsInterpreted = false;
```

Comment:
Checking code to convince myself that `comp` can not be `NULL`. I see `TR_ResolvedRelocatableJ9Method` is constructed here : https://github.com/eclipse/openj9/blob/b5e3c2a4c84e4c4966e1568aa7349b28a6bbbb88/runtime/compiler/env/j9method.cpp#L1353 . 
It constructs `TR_ResolvedRelocatableJ9Method` and validates the class if `comp` is not `NULL` but as `TR_ResolvedRelocatableJ9Method` is already constructed, it may be possible that `isInterpreted()` on `TR_ResolvedRelocatableJ9Method` is called even when `comp` is `NULL`. I think it is safe to have null check here to accidentally cause segmentation fault.